relay-general/src/protocol/contexts/app.rs

```diff
@@ -36,6 +36,9 @@ pub struct AppContext {
     /// Amount of memory used by the application in bytes.
     pub app_memory: Annotated<u64>,
 
+    /// A flag indicating whether the app is in foreground or not. An app is in foreground when it's visible to the user.
+    pub app_in_foreground: Annotated<bool>,
```

Comment:
nit: I realize it's consistent with the other fields in this context, but the `app_` prefix seems redundant to me.

Reply:
Yeah, I prefer to not have the `app_` prefix as well. It might not be consistent within the app context, but looking at other contexts, it seems to be a better fit without the prefix. Let me remove the prefix 👍